perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -314,7 +315,8 @@ def PrepareVMEnvironment(self):
       if self.is_static:
         self.SnapshotPackages()
       self.SetupPackageManager()
-      self.InstallPackages('python')
+      if (self.OS_TYPE != 'clear'):
+        self.InstallPackages('python')
```

Comment:
I think it would be nicer to add a SwupdInstall in linux_packages/python. Even if it is unimplemented and simply logs a warning.

We do need this for most benchmarks. Is there a reason we can't install Python for Clear, or is this just left for future work. Leaving it for future work is fine.

Reply:
@s-deitz done. Changed with `self.Install('python')` here. Also added `Swupdinstall` to linux_packages/python and the `python` package to `AptInstall`, so that the behavior remains the same as before.